draft-ietf-quic-transport.md

```diff
@@ -4111,10 +4112,10 @@ address of the client; see {{address-validation}}.
 
 The Path Maximum Transmission Unit (PMTU) is the maximum size of the entire IP
 packet including the IP header, UDP header, and UDP payload.  The UDP payload
-includes QUIC packet headers, protected payload, and any authentication fields.
-The PMTU can depend on path characteristics, and can therefore change over time.
-The largest UDP payload an endpoint sends at any given time is referred to as
-the endpoint's maximum datagram size.
+includes one or more QUIC packet headers, the protected payloads, and any
+authentication fields.  The PMTU can depend on path characteristics, and can
```

Comment:
I think that we can drop "authentication fields" as that is just part of "protected payload".